sdk/eventhubs/azure_messaging_eventhubs/src/consumer/mod.rs

```diff
@@ -24,21 +21,25 @@ use azure_core_amqp::{
     cbs::{AmqpClaimsBasedSecurity, AmqpClaimsBasedSecurityApis},
     connection::{AmqpConnection, AmqpConnectionApis, AmqpConnectionOptions},
     management::{AmqpManagement, AmqpManagementApis},
-    messaging::{AmqpDeliveryApis, AmqpSource, AmqpSourceFilter},
+    messaging::{AmqpSource, AmqpSourceFilter},
     receiver::{AmqpReceiver, AmqpReceiverApis, AmqpReceiverOptions, ReceiverCreditMode},
     session::{AmqpSession, AmqpSessionApis},
     value::{AmqpDescribed, AmqpOrderedMap, AmqpSymbol, AmqpValue},
 };
-use futures::stream::Stream;
+use event_receiver::EventReceiver;
 use std::{
     collections::HashMap,
     default::Default,
     fmt::Debug,
     sync::{Arc, OnceLock},
+    time::Duration,
 };
 use tracing::{debug, trace};
 use url::Url;
 
+/// Receive messages from a partition.
+pub mod event_receiver;
```

Comment:
We can't just fold this into the `consumer` module? We don't want super-long module paths.

Reply:
Ah - the objection is to the `pub`, not to the `mod`.